base.php

```diff
@@ -418,7 +418,8 @@ class Entry
         Book::ALL_RECENT_BOOKS_ID    => 'images/recent.png',
         Tag::ALL_TAGS_ID             => 'images/tag.png',
         Language::ALL_LANGUAGES_ID   => 'images/language.png',
-        CustomColumn::ALL_CUSTOMS_ID => 'images/tag.png',
+        CustomColumn::ALL_CUSTOMS_ID => 'images/custom.png',
+        "cops:rating$"               => 'images/rating.png',
```

Comment:
That strange I tried to replace this line by `Rating::ALL_RATING_ID => 'images/rating.png'`